.gitignore

```diff
@@ -187,5 +187,9 @@ fabric.properties
 ### VSCode ###
 **/.vscode/
 
+pyrightconfig.json
```

Comment:
It seems to me like this ought to be committed and part of the repo – am I mistaken?

Reply:
This points to a local .venv typically, which is specific to user setup. If we wanted to configure pyright to enforce rules in a custom way, we can do that right in pyproject.toml